gmsh_utils/gmsh_IO.py

```diff
@@ -154,103 +155,182 @@ def create_surface(self, line_ids: Union[List[int], npt.NDArray[np.int_]], name_
-            self.create_point(coordinate, element_size)
+            point_id = self.create_point(point, default_mesh_size)
+            list_point_ids.append(point_id)
+        return list_point_ids
 
-        line_lists = []
+    def make_lines(self, point_pairs: Union[List[List[int]], npt.NDArray[np.int_]]) \
+            -> Union[List[int], npt.NDArray[np.int_]]:
+        """
+        Makes lines with line tags by getting point pairs.
 
-        for i in range(len(point_pairs)):
-            line = [point_pairs[i][0], point_pairs[i][1]]
-            line_lists.append(i + 1)
-            self.create_line(line)
+        Args:
+            point_pairs (Union[List[List[int]], npt.NDArray[np.int_]]): A list of pairs of point tags which create a
+            line.
 
-        surface_id: int = self.create_surface(line_lists, name_label)
-        return surface_id
+        Returns:
+            List[List[int]]: A list of line tags.
+        """
 
-    def make_geometry_3D(self, point_coordinates: Union[List[List[float]], npt.NDArray[np.float64]],
-                         point_pairs: Union[List[List[int]], npt.NDArray[np.int_]],
-                         mesh_size: float, extrusion_length: Union[List[float],
-                                                                   npt.NDArray[np.float64]], name_label: str) -> None:
+        list_lines = []
+        for point_pair in range(len(point_pairs)):
+            line = [point_pairs[point_pair][0], point_pairs[point_pair][1]]
+            line_id = self.create_line(line)
+            list_lines.append(line_id)
+        return list_lines
+
+    def make_surfaces(self, line_list: Union[List[int], npt.NDArray[np.int_]], name_label: str) \
+            -> Union[List[int], npt.NDArray[np.int_]]:
         """
-        Creates 3D geometries by extruding the 2D surface
+        Makes surfaces with surface tags by getting line tags.
 
         Args:
-            point_coordinates (Union[List[float], npt.NDArray[float]]): Geometry points coordinates.
-            point_pairs (Union[List[List[int]], npt.NDArray[npt.NDArray[int]]]): A list of point tags of two consecutive
-                points in an array.
-            mesh_size (float): Mesh size.
-            extrusion_length (Union[List[float], npt.NDArray[float]]): The extrusion length in x, y and z direction.
-            name_label (str): surface name label from user input.
+            line_list (Union[List[int], npt.NDArray[np.int_]]): A list of line tags in order.
+            name_label (str): surface name labels provided by user input.
+
+        Returns:
+            List[int]: A list of surface tags.
+        """
+
+        surfaces = []
+        surfaces.append(self.create_surface(line_list, name_label))
+
+        return surfaces
+
+    def make_volume(self, surface_id: Union[List[int], npt.NDArray[np.int_]],
+                    extrusion_length: Union[List[float], npt.NDArray[np.float64]], name_label: str) -> None:
```

Comment:
```suggestion
                    extrusion_length: Union[List[float], npt.NDArray[np.float64]], name_labels: List[str]) -> None:
```

Reply:
since it is doing it layer by layer, it only gets the name label for that layer therefore it's string